tests/test_target_order.py

```diff
@@ -697,7 +730,7 @@ def should_include_target(target: Target):
             return False
 
         # Finally, filter out dependencies of any of the above:
-        if issubclass(cls, BuildGmp):
+        if issubclass(cls, BuildGmp) or issubclass(cls, BuildMpfr):
```

Comment:
```suggestion
        if issubclass(cls, (BuildGmp, BuildMpfr)):
```
Polymorphic Python magic to short it (and match style elsewhere in the function)